website/docs/downloads/windows.md

```diff
@@ -25,6 +25,21 @@ the changes and improvements introduced by this release.
     If Windows 8.x or Windows 10 prevents you from running DOSBox Staging via
     Microsoft Defender SmartScreen, see the [section below](#microsoft-defender-smartscreen).
 
+For 32-bit Windows 7 and newer, and for 64-bit Windows 7 hosts use the respective MSVC builds:
+- The 32-bit MSVC build [DOSBox Staging 0.80.1 32-bit (zip)][0_80_1_x32_ZIP]
+- The 64-bit MSVC build on Windows 7 or newer can be optionally selected in the installer.
+
+Windows Vista hosts are supported out-of-the-box up to release 0.78.1.
+<br>
+Subsequent releases can be [build from source][5] by excluding FluidSynth (MIDI) and Slirp (NE2000) via:
+
+!!! MSYS2 Clang build with step 8 command 
+
+    meson setup -Duse_fluidsynth=false -Duse_slirp=false build/release-clang --native-file=.github/meson/native-clang.ini
+
+[5]:https://github.com/dosbox-staging/dosbox-staging/blob/main/docs/build-windows.md#build-using-msys2 
+
+CPUs without SSE4.2 are supported up to release 0.75.0. 
```

Comment:
General comment: please fix up the formatting once you're done with the above.

<img width="801" alt="image" src="https://user-images.githubusercontent.com/698770/217957196-50faa9e1-171d-4a4b-9f4f-bea4c1a186ed.png">

Reply:
additions reworded and reformatted